fog/uri/src/lib.rs

```diff
@@ -129,6 +161,24 @@ mod tests {
             ResponderId::from_str("node1.test.mobilecoin.com:666").unwrap()
         );
         assert!(!uri.use_tls());
+
+        let uri =
+            FogViewRouterUri::from_str("fog-view-router://node1.test.mobilecoin.com:443/").unwrap();
```

Comment:
Optional/Equivalent: Consider omitting the port here, to exercise the default port behaviour.

```suggestion
            FogViewRouterUri::from_str("fog-view-router://node1.test.mobilecoin.com/").unwrap();
```

Reply:
All of the other test cases keep the port so I'll leave it. We can fix this in a follow up PR.